src/main/webapp/js/user-page-loader.js

```diff
@@ -77,7 +77,8 @@ function buildMessageDiv(message) {
   const headerDiv = document.createElement('div');
   headerDiv.classList.add('message-header');
   headerDiv.appendChild(document.createTextNode(
-      message.user + ' - ' + new Date(message.timestamp)));
+      message.user + ' - ' + new Date(message.timestamp) + 
```

Comment:
This stickler message I believe wants you to convert this to ES6 template literals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

So this would look like `${message.user} - ${new Date(message.timestamp)} [${message.sentimentScore}]`
I believe

Reply:
I actually didn't write this code...I tried changing it to that and it gave me a bunch of errors saying I was missing comments and stuff...any ideas? @Tetheta